nessie/src/main/java/org/apache/iceberg/nessie/NessieIcebergClient.java

```diff
@@ -142,15 +148,27 @@ private UpdateableReference loadReference(String requestedRef, String hash) {
   }
 
   public List<TableIdentifier> listTables(Namespace namespace) {
+    return listContents(namespace, Content.Type.ICEBERG_TABLE);
+  }
+
+  public List<TableIdentifier> listViews(Namespace namespace) {
+    return listContents(namespace, Content.Type.ICEBERG_VIEW);
+  }
+
+  /** Lists Iceberg table or view from the given namespace */
+  protected List<TableIdentifier> listContents(Namespace namespace, Content.Type type) {
```

Comment:
is there any value in keeping this (and others) protected? I would have thought it should be ok to make this private, since all the other places would then use the table/view-specific methods

Reply:
In the `NessieCatalog` itself I have some common code for content which calls these protected classes. 

If I doesn't have common code in `NessieCatalog`, it will be code duplication. 
For example see, `NessieCatalog.renameContent()`

Reply:
`NessieCatalog.table --> NessieCatalog.content --> NessieClient.content` this way it is less code duplication. 

`NessieCatalog.table -->  NessieClient.table --> NessieClient.content` will cause duplicate lines of code in  `NessieCatalog`